imported cinemachine package

BenicioLadines opened the pull request:
testing out cinemachine i think itll work really well with a freelook camera attached to our player

BenicioLadines committed:
imported cinemachine package

testing out cinemachine i think itll work really well with a freelook camera

BenicioLadines committed:
Merge branch 'develop' into mouseCamera

TheStress reviewed (commented):
Have the mouse disapear when controlling the camera and make sure it works for controller

BenicioLadines committed:
camera aim works with nelsons dash!

the camera still needs work but I made it so the PlayerInput script gets vector3 direction oriented towards the camera's rotation (cred to whoever wrote playerMovement)

BenicioLadines committed:
added right stick support and locked mouse

switch between controller n keyboard/mouse with the enum state i put in playerInput

TheStress reviewed (approved):
Looks good
